### ISSUE TITLE
I can't load big mp4 file(> 80Mb) on react-native-video 5.0.2

### ISSUE THREAD

**Issue by masterdev1027** [TheWidlarzGroup/react-native-video#1784]
Hi, DevTeam
I faced the issue while I'm loading video file which is more than 80Mb from AWS server.
I can't load video file with black screen on react-native-video 5.0.2 in react-native-0.61.1
Also I tried to load 4Mb video file from AWS, but it worked fine.
My app which was on react-native-0.59 used react-native-video 4.4 version before, at that time it worked nicely but new version didn't work.
Should I set additional config on 5.0.2 version?

Regards.

**Comment by CHaNGeTe** [comment]
Is your AWS instance configured to handle progressive requests?
Accept-Ranges, Range, and so on?

**Comment by trickeyd** [comment]
I'm having a similar problem loading from gcloud bucket. When I updated to iOS 13 everything just stopped working. The only files I can download seem to be ones less than 1MB. Everything else is the same.

I've tried a few versions of react-native-video, but to no avail. The errors I get keep changing too, seemingly at random. I get 1001, 1011, and 1201

Any thoughts at all?

**Comment by CHaNGeTe** [comment]
If you can provide stream urls will help.
If you can't, calling your stream url in postman with header: Range bytes=0-2 will let you know if your server is configured for progressive download.
**Status code 206** - Partial content (and receiving header: Content-Length (3) and content-range (bytes 0-2/SIZE OF THE FILE) --> configured for progressive
**Status code 200** - Does not support progressive

**Comment by trickeyd** [comment]
Hey thanks for your response.

It is defiantly configured for progressive as the same files were working fine prior to iOS 13 update. In postman, 206 Partial Content is returned as expected.

I made a test URL which exhibits the same problem:
https://storage.googleapis.com/gobmob_test_video/100/footie.mp4

Many thanks!

**Comment by trickeyd** [comment]
Hi @CHaNGeTe,

did you get a chance to investigate the sample I sent you by any chance?

We have changed a bunch of our videos to Cloudinary as that works fine, but to change them all would be a huge job, so I'd love a quick fix in the meantime, but I don't know where to go from here as I have checked that encoding, headers and metadata are the same, an it is only iOS 13. I'm at a bit of a loss.

Cheers!
